server/commands.py

```diff
@@ -3688,7 +3687,7 @@ def ooc_cmd_passage_clear(client: ClientManager.Client, arg: str):
     /passage_clear <area_range_start>, <area_range_end>
 
     /passage_restore <area_range_start>, <area_range_end>
```

Comment:
I know this wasn't you, but could you please take out this improper syntax line for passage clear?